mycpp/gc_heap.cc

```diff
@@ -173,7 +173,7 @@ void Heap::Collect() {
           case Tag::Scanned:
             break;
           default:
-            assert(0);
+            assert(0); // NOTE(Jesse): Pretty sure this is InvalidCodePath();
```

Comment:
I think this whole TODO_BUG block is hit by a bug in a test, probably mycpp/examples.

It's not enabled right now but we will probably hit it as we test the collector.